### PR TITLE
Categorize plugins

### PR DESCRIPTION
**Pull request by blyedev**
> Which issue, if any, is this issue related to?

#80

> Is there anything in the PR that needs further explanation?

I could use input on the categories, this is merely a first draft. Even in this state though I find it substantially better than the status quo

### CODE REVIEW

**Comment by ybiquitous** [comment]
Thanks, I love this categorizing. 👍🏼 

Can you resolve conflicts?

**Comment by blyedev** [comment]
> Thanks, I love this categorizing. 👍🏼
> 
> Can you resolve conflicts?

Done

**Comment by blyedev** [comment]
Closes #80

**Review by ybiquitous — CHANGES_REQUESTED** [review]
Can you sort plugins alphabetically?

**Comment by blyedev** [comment]
> Can you sort plugins alphabetically?

Done

**Comment by blyedev** [comment]
Actually wait I just realized I messed up the original rebase that's why I still had conflicts, let me rewrite history so it's cleaner

**Comment by ybiquitous** [comment]
Ah, keeping the commits as-is is fine. We will squash and merge PRs.

File: README.md [hunk 1]
```diff
@@ -82,48 +82,77 @@
...
+- [stylelint-force-app-name-prefix](https://www.npmjs.com/package/stylelint-force-app-name-prefix) - Enforce app name prefix for rules.
+- [stylelint-max-lines](https://www.npmjs.com/package/stylelint-max-lines) - Limit the number of lines in a source.
 - [stylelint-order](https://www.npmjs.com/package/stylelint-order) - Enforce a specific order for things like properties (Pack).
-- [stylelint-plugin-defensive-css](https://www.npmjs.com/package/stylelint-plugin-defensive-css) - Enforce defensive CSS best practices (Pack).
-- [stylelint-plugin-logical-css](https://www.npmjs.com/package/stylelint-plugin-logical-css) - Enforce logical CSS properties, values and units (Pack).
 - [stylelint-react-native](https://www.npmjs.com/package/stylelint-react-native) - Enforce React Native rules (Pack).
+- [stylelint-transform-function-no-whitespace](https://www.npmjs.com/package/stylelint-transform-function-no-whitespace) - Disallow whitespace between transform functions and their parentheses.
+
+### Nesting, and specificity
```
Comment (ybiquitous) [inline README.md:114]: [suggestion] How about trimming a comma?

```suggestion
### Nesting and specificity
```

**Comment by ybiquitous** [comment]
Thank you, overall LGTM 👍🏼 

@jeddy3 If you're interested and have some time, please provide feedback.

**Comment by blyedev** [comment]
> Ah, keeping the commits as-is is fine. We will squash and merge PRs.

It's a personal preference so that I can review my own commit history properly, so excuse all the force pushes its my first time contributing to two pr's on one fork.

Otherwise all requested changes are fixed, added 2 fixes too

**Review by jeddy3 — APPROVED** [review]
@blyedev Thanks for the pull request and for doing the bulk of the work.

Categorisation is hard, and some plugin packs don't fit neatly into one category, so we've avoided doing it so far. However, it is an improvement for the user. And, going forward, plugin authors can place their plugins into the most appropriate category.

I've pushed a commit to move a few things around based on what I know about the plugins. It looks good enough to me.

**Comment by blyedev** [comment]
> @blyedev Thanks for the pull request and for doing the bulk of the work.
> 
> Categorisation is hard, and some plugin packs don't fit neatly into one category, so we've avoided doing it so far. However, it is an improvement for the user. And, going forward, plugin authors can place their plugins into the most appropriate category.
> 
> I've pushed a commit to move a few things around based on what I know about the plugins. It looks good enough to me.

Glad to help and thank you for the incredible work on stylelint. It really is a great tool and I cannot believe how underutilized it is.

**Review by ybiquitous — APPROVED** [review]
Thank you, this is now much improved! 👍🏼

**Pull request merged**